File: 1.Bolum/5-fibonnaci/c/main.c
main()
{
    fibonnaci();
}
int fibonnaci()
{
    static int f1 = 5, f2 = 11;
    int g;
    g = f1 + f2;
    f1 = f2;
    f2 = g;
    printf("%d", g);
}

File: 1.Bolum/5-fibonnaci/go/main.go
package main

import "fmt"

func main() {
	fibonnaci()
}

func fibonnaci() {
	var f1, f2 int = 5, 11
	var g int
	g = f1 + f2
	f1 = f2
	f2 = g
	fmt.Printf("%d", g)
}
